core/templates/dev/head/css/oppia.css

```diff
@@ -3583,11 +3583,13 @@ md-card.preview-conversation-skin-supplemental-card {
 .oppia-dashboard-container a:hover,
 .oppia-dashboard-container a:active,
 .oppia-dashboard-container a:visited {
-  color: #009688;
+  color: inherit;
+  text-decoration: none;
 }
 
-.oppia-dashboard-container a:hover {
-    text-decoration: underline;
+.oppia-dashboard-list-summary {
+  display:inline-block;
+  width: 100%;
```

Comment:
Though I'm not sure.. but you could try using `display: block;` instead of `display: inline-block`.. so that there might not be a need to use `width: 100%;` (I'm really not sure about this one, but do give it a try and see if it works!)